lib/common.sh

```diff
@@ -207,8 +210,17 @@ fi
 # Enable ironic restart feature when the TLS certificate is updated
 export RESTART_CONTAINER_CERTIFICATE_UPDATED=${RESTART_CONTAINER_CERTIFICATE_UPDATED:-${IRONIC_TLS_SETUP}}
 
-# Baremetal operator image
-export BAREMETAL_OPERATOR_IMAGE=${BAREMETAL_OPERATOR_IMAGE:-"${CONTAINER_REGISTRY}/metal3-io/baremetal-operator"}
+# Baremetal operator and Ironic image
+# Set the tags based on the CAPM3RELEASE. Both BMO and Ironic has tags with the
+# following format: capm3-vX.Y.Z, where vX.Y.Z is the CAPM3 release.
+bmo_tag="latest"
+ironic_tag="latest"
+if [ -n "${CAPM3RELEASE}" ]; then
+  bmo_tag="capm3-${CAPM3RELEASE}"
+  ironic_tag="capm3-${CAPM3RELEASE}"
+fi
```

Comment:
As far as I can tell `CAPM3RELEASE` will always be defined at this point. Should we just skip the if statement?
Another question would be if we want to use the latest tag for some tests? Now it will always use a tag like `capm3-v1.1.0` except for BMO PRs where it will build the image locally.

Reply:
CAPM3RELEASE should be always there since you are sourcing releases.sh meaning it will always override the previous  bmo_tag/ironic_tag vars, so I think we can skip the condition. 



> nother question would be if we want to use the latest tag for some tests?

if that is the case, depending on what kind of test it is, we can introduce condition so that:

```
if X-test; then
    bmo_tag="latest"
    ironic_tag="latest"
else
    bmo_tag="capm3-${CAPM3RELEASE}"
    ironic_tag="capm3-${CAPM3RELEASE}"
fi

```

Reply:
Thanks for confirming. I think I'll just remove the if statement then.

Btw, I put the tag variables in lowercase here (`bmo_tag` and `ironic_tag`) as I wanted to make it clear they are not to be set by the user. If there is already some other convention for this please let me know.

Reply:
> Now it will always use a tag like `capm3-v1.1.0` except for BMO PRs where it will build the image locally.

+ for Ironic